extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/HttpAuthorizer.java

```diff
@@ -20,7 +20,7 @@
  * Class that is responsible for running the HTTP based permission checks
  */
 @Singleton
-public class HttpAuthorizer extends AbstractHttpAuthorizer {
+public final class HttpAuthorizer extends AbstractHttpAuthorizer {
```

Comment:
I would say the same as for the HttpAuthenticator - but because there is an abstract class, it should be fine right? The user can still extends the abstract class.

Reply:
Same answer as above. I don't need to do it on existing class if you disagree with arguments.